src/AspNet.Security.OAuth.Zalo/ZaloAuthenticationExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNet.Security.OAuth.Zalo
+{
+    public static class ZaloAuthenticationExtensions
+    {
+        /// <summary>
+        /// Adds <see cref="ZaloAuthenticationHandler"/> to the specified
+        /// <see cref="AuthenticationBuilder"/>, which enables Zalo authentication capabilities.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <returns>The <see cref="AuthenticationBuilder"/>.</returns>
+        public static AuthenticationBuilder AddZalo(
+            [NotNull] this AuthenticationBuilder builder)
+        {
+            return builder.AddZalo(options => { });
+        }
+
+        /// <summary>
+        /// Adds <see cref="ZaloAuthenticationHandler"/> to the specified
+        /// <see cref="AuthenticationBuilder"/>, which enables Zalo authentication capabilities.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <param name="configuration">The delegate used to configure the Gitter options.</param>
+        /// <returns>The <see cref="AuthenticationBuilder"/>.</returns>
+        public static AuthenticationBuilder AddZalo(
+            [NotNull] this AuthenticationBuilder builder,
+            [NotNull] Action<ZaloAuthenticationOptions> configuration)
+        {
+            return builder.AddZalo(
+                ZaloAuthenticationDefaults.AuthenticationScheme,
+                ZaloAuthenticationDefaults.DisplayName,
+                configuration);
+        }
+
```

Comment:
The other providers provider a 4th extension method. For consistency can you add the equivalent please?

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/542ac858ea1d1be1ca6fab2dacb37d4fd309248f/src/AspNet.Security.OAuth.Amazon/AmazonAuthenticationExtensions.cs#L44-L57